examples/README.md

```diff
@@ -20,3 +20,37 @@ Additional examples for [Confluent Cloud](https://www.confluent.io/confluent-clo
 
 * [confluent_cloud.py](confluent_cloud.py): produces messages to Confluent Cloud and then reads them back again
 * [confluentinc/examples](https://github.com/confluentinc/examples/tree/master/clients/cloud/python): integrates Confluent Cloud and Confluent Cloud Schema Registry
+
+## venv setup
+
+It's usually a good idea to install Python dependencies in a virtual environment to avoid
+conflicts between projects.
+
+To setup a venv with the latest release version of confluent-kafka and dependencies of all examples installed:
+
+```
+> python3 -m venv venv_examples
```

Comment:
Never seen '>' denote the standard bash PS1 prompt, that's usually the PS2 (multiline) prompt. Use `$ ` instead